addons/website_sale_loyalty/views/website_sale_templates.xml

```diff
@@ -36,8 +36,24 @@
                                         <input type="hidden" name="reward" t-att-value="reward.id"/>
                                         <div class="alert alert-success text-start mt16" role="alert">
                                             <div class="d-flex flex-row">
-                                                <div class="flex-grow-1">
+                                                <div class="flex-grow-1 text-break">
                                                     <strong t-esc="reward.description"/>
+                                                    <div t-if="reward.reward_type == 'product'"
+                                                        class="mt-1 pe-3">
+                                                            <select t-if="reward.multi_product"
+                                                                class="o_select w-100 form-select form-select-sm css_attribute_select"
+                                                                name="product">
```

Comment:
```suggestion
                                                                name="product_id">
```
Since you send a product id, let's name the variable accordingly.